src/explore-education-statistics-frontend/src/hooks/useCookies.ts

```diff
@@ -106,5 +131,14 @@ export function useCookies(initialCookies?: Dictionary<string>) {
         enableGoogleAnalytics();
       }
     },
+    setUserTestingBannerDisabledCookie(isDisabled: boolean) {
+      const value = isDisabled ? 'true' : 'false';
+
+      setCookie(
+        allowedCookies.disableUserTestingBanner.name,
+        value,
+        allowedCookies.disableUserTestingBanner.options,
+      );
+    },
```

Comment:
This was missing from the `onClick` of the close button in `userTestingBanner`. I initially assumed we'd want to set a cookie so that the banner doesn't show up all the time. Might be wrong though

Reply:
It looks like the `disableUserTestingBanner` isn't actually used at all after it's set. I think we just need the one `userTestingBannerSeen` cookie.

Reply:
Ah yep, I was mixing the two up. I've removed it now